actors/user/src/test/java/org/sunbird/user/EducationManagementActorTest.java

```diff
@@ -0,0 +1,168 @@
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.datasecurity.EncryptionService;
+import org.sunbird.common.request.Request;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.user.actors.EducationManagementActor;
+import org.sunbird.user.dao.impl.AddressDaoImpl;
+import org.sunbird.user.dao.impl.EducationDaoImpl;
+import org.sunbird.user.util.UserActorOperations;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({
+  ServiceFactory.class,
+  ElasticSearchUtil.class,
+  EncryptionService.class,
+  EducationDaoImpl.class,
+  AddressDaoImpl.class
+})
+@PowerMockIgnore({"javax.management.*"})
+public class EducationManagementActorTest {
+
+  private static final ActorSystem system = ActorSystem.create("system");
+  private static final Props props = Props.create(EducationManagementActor.class);
+  private static final CassandraOperation cassandraOperation = mock(CassandraOperationImpl.class);
+
+  @BeforeClass
+  public static void beforeEachTest() throws Exception {
+
+    PowerMockito.mockStatic(ServiceFactory.class);
+    when(ServiceFactory.getInstance()).thenReturn(cassandraOperation);
+    when(cassandraOperation.insertRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyMap()))
+        .thenReturn(getSuccessResponse());
+    when(cassandraOperation.deleteRecord(
+            Mockito.anyString(), Mockito.anyString(), Mockito.anyString()))
+        .thenReturn(getSuccessResponse());
+  }
+
+  @Test
+  public void testInsertEducation() {
+
+    boolean result =
+        testScenario(UserActorOperations.INSERT_USER_EDUCATION, true, getUpsertResponse(), true);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testUpdateEducationForDeleteEducationDetailsSuccess() {
+
+    boolean result =
+        testScenario(UserActorOperations.UPDATE_USER_EDUCATION, true, getUpsertResponse(), true);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testUpdateEducationWithUpsertEducationDetailsSuccess() {
+
+    boolean result =
+        testScenario(UserActorOperations.UPDATE_USER_EDUCATION, false, getUpsertResponse(), true);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testInsertEducationFailure() {
+
+    boolean result =
+        testScenario(UserActorOperations.INSERT_USER_EDUCATION, true, new Response(), false);
+    assertTrue(result);
+  }
+
+  @Test
+  public void testUpdateEducationForDeleteEducationDetailsFailure() {
+
+    boolean result =
+        testScenario(UserActorOperations.UPDATE_USER_EDUCATION, true, new Response(), false);
+    assertTrue(result);
+  }
+
+  private boolean testScenario(
+      UserActorOperations actorOperation, boolean isDelete, Response response, boolean isSuccess) {
+    return testScenario(actorOperation, false, isDelete, response, false);
+  }
+
+  private boolean testScenario(
+      UserActorOperations operation,
+      boolean isParamReq,
+      boolean isDelete,
+      Response response,
+      boolean isSuccess) {
```

Comment:
what is the purpose of overload ? 
it seems you are calling it from only one place.